test/e2e/metamask-ui.spec.js

```diff
@@ -37,6 +40,37 @@ describe('MetaMask', function () {
       'test-dapp',
       'dist',
     );
+    fixtureServer = new FixtureServer();
```

Comment:
Do we still need these changes, now that the initial controller state has been updated?

Reply:
Addressed in [61d22a6](https://github.com/MetaMask/metamask-extension/pull/19486/commits/61d22a697528f6ccb84938e8bd78deef8082516a).